Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/DictationInputController.cs

```diff
@@ -274,13 +303,14 @@ private static void DictationRecognizer_DictationError(string error, int hresult
         {
             dictationResult = $"{error}\nHRESULT: {hresult}";
 
-            InputSystem.RaiseDictationError(source, dictationResult);
+            InputSystem.RaiseDictationError(inputSource, dictationResult);
             textSoFar = null;
             dictationResult = string.Empty;
         }
 
-#endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
 
         #endregion Dictation Recognizer Callbacks
+#endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+        #endregion UWP and Windows Standalone Implementations
```

Comment:
We can get rid of this region, seeing as each controller should be platform specific.

Reply:
Each implementation is a platform